docs/querying/functions.md

```diff
@@ -380,15 +380,17 @@ do not show up in the returned vector.
 Similarly, `histogram_stdvar(v instant-vector)` returns the estimated standard
 variance of observations in a native histogram.
 
-## `holt_winters()`
+## `exponential_smoothing()`
```

Comment:
Let's call it `double_exponential_smoothing` and link to https://www.itl.nist.gov/div898/handbook/pmc/section4/pmc433.htm  in the documentation below?

Reply:
The link lgtm. Wikipedia would be ok to link to as well imo, but its no significant improvement.